<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GeneralSystemAdministrationWidget</class>
 <widget class="QWidget" name="GeneralSystemAdministrationWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>417</width>
    <height>360</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="commonGroupBox">
     <property name="title">
      <string>Administration</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,0">
      <item>
       <widget class="QWidget" name="widget" native="true">
        <layout class="QVBoxLayout" name="verticalLayout_5">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QPushButton" name="businessRulesButton">
           <property name="text">
            <string>&amp;Alarm/Event Rules</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="eventLogButton">
           <property name="text">
            <string>&amp;Event Log</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="cameraListButton">
           <property name="text">
            <string>&amp;Camera List</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="healthMonitorButton">
           <property name="text">
            <string>&amp;Health Monitoring</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="camerasGroupBox">
     <property name="title">
      <string>Camera Management</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QnCameraManagementWidget" name="cameraWidget" native="true"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="backupGroupBox">
     <property name="title">
      <string>System Configuration Backup and Restore</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QnDatabaseManagementWidget" name="backupWidget" native="true"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QnCameraManagementWidget</class>
   <extends>QWidget</extends>
   <header location="global">ui/widgets/settings/camera_management_widget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>QnDatabaseManagementWidget</class>
   <extends>QWidget</extends>
   <header location="global">ui/widgets/settings/database_management_widget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
